activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Update `open_ended?` method in `Predications` to include `nil?` check now that ruby supports both endless ranges in Ruby 2.6 and beginless ranges in Ruby 2.7.
```

Comment:
Can you remove the CHANGELOG entry? Arel is not part of the public API of Rails so there is no need for changelogs entries for its changes.

Reply:
Yeah I was wondering about that since it is private, makes perfect sense!